os/include/netdb.h

```diff
@@ -323,6 +327,10 @@ int getaddrinfo(const char *host, const char *service, const struct addrinfo *hi
 
 /**
  * @brief freeaddrinfo() frees the memory allocated by the getaddrinfo() function.
+ *
+ * @details @b #include <netdb.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised